### ISSUE TITLE
Tests Failing

### ISSUE THREAD

**Issue by jamesward** [apple/pkl#19]
On `main` 5 tests are failing. Not sure if there is something else I need to setup. Full log (too long to put in a comment):
https://gist.github.com/jamesward/738f703601fdbac2ef64fa18ef3dd3ef

**Comment by bioball** [comment]
What OS are you on, and what is your java version?

**Comment by jamesward** [comment]
JDK 17. Ubuntu Linux.

**Comment by bioball** [comment]
Hm, we test on that. Can you try adding env var `LANG=en_US.UTF-8`?

**Comment by jamesward** [comment]
Same failures with that env var.

**Comment by holzensp** [comment]
This appears to have been caused (somehow) by the bump to Gradle 7.6, which has been reverted. Can you confirm these tests no longer fail for you?

**Comment by bioball** [comment]
I don't think it was actually; James tested prior to us merging the Gradle verison bump
